src/codegen/revert.rs

```diff
@@ -38,15 +39,21 @@ pub enum SolidityError {
     String(Expression),
```

Comment:
Why is this called `String` and not `Error`?

Reply:
If you write `revert("foo")` then this is just a string. However in the catch clause it's `Error(string)`. I'm not really sure which is ultimately the correct one. What I'm sure about is that I don't care how this is called, I can change it to `Error` if you want.